cogs/sudo.py

```diff
@@ -279,74 +235,61 @@ def get_size(bytes, suffix="B"):
-                      • Total Bytes Received: **{get_size(net_io.bytes_recv)}**
-                    """
-
-        embed = discord.Embed(title="BOT NET STATS", color=discord.Color.red())
+        netio_stats = textwrap.dedent(f"""
+                                      • Total Bytes Sent: **{self.get_size(net_io.bytes_sent)}**
+                                      • Total Bytes Received: **{self.get_size(net_io.bytes_recv)}**
+        """)
 
+        embed = Embed(title="BOT NET STATS", color=Color.red())
         embed.add_field(name="**❯❯ Net Interface Stats**", value=net_interfaces, inline=False)
         embed.add_field(name="**❯❯ Net IO Stats**", value=netio_stats, inline=False)
-
-        embed.set_author(name=f"{self.client.user.name}'s Network Info", icon_url=self.client.user.avatar_url)
+        embed.set_author(name=f"{self.bot.user.name}'s Network Info", icon_url=self.bot.user.avatar_url)
 
         await ctx.send(embed=embed)
 
     @sudo.command()
-    @commands.check(check)
-    async def gpuinfo(self, ctx):
+    async def gpuinfo(self, ctx: Context) -> None:
+        """Get detailed GPU info."""
+        embed = Embed(title="BOT GPU INFO", color=Color.red())
 
-        await ctx.send("**" + "=" * 15 + "GPU Details" + "=" * 15 + "**")
-        gpus = GPUtil.getGPUs()
+        for gpu in GPUtil.getGPUs():
+            gpu_details = textwrap.dedent(f"""
+                                          • Load: {gpu.load*100}%
+                                          • Temperature: {gpu.temperature} °C
 
-        list_gpus = []
+                                          • Free Memory: {gpu.memoryFree}MB
+                                          • Used Memory: {gpu.memoryUsed}MB
+                                          • Total Memory: {gpu.memoryTotal}MB
 
-        for gpu in gpus:
-            # get the GPU id
-            gpu_id = gpu.id
+                                          • UUID: {gpu.uuid}
+            """)
+            embed.add_field(name=f"**❯❯ GPU: {gpu.name}({gpu.id})**", value=gpu_details, inline=False)
 
-            # name of GPU
-            gpu_name = f"({gpu_id}){gpu.name}"
-
-            # get % percentage of GPU usage of that GPU
-            gpu_load = f"{gpu.load * 100}%"
-
-            # get free memory in MB format
-            gpu_free_memory = f"{gpu.memoryFree}MB"
-
-            # get used memory
-            gpu_used_memory = f"{gpu.memoryUsed}MB"
-
-            # get total memory
-            gpu_total_memory = f"{gpu.memoryTotal}MB"
-
-            # get GPU temperature in Celsius
-            gpu_temperature = f"{gpu.temperature} °C"
-
-            gpu_uuid = gpu.uuid
-            list_gpus.append((
-                gpu_name, gpu_load, gpu_free_memory, gpu_used_memory,
-                gpu_total_memory, gpu_temperature, gpu_uuid
-            ))
-
-        await ctx.send(tabulate(list_gpus, headers=("name", "load", "free memory", "used memory", "total memory",
```

Comment:
Change it to an embed soon.

Reply:
As described before, this was done already be the first commit f2b0e92 you've highlighted a removed part of the code, that's why it's red.